registry/api/errcode/errors.go

```diff
@@ -97,25 +101,28 @@ type Error struct {
 
 var _ error = Error{}
 
-// ErrorCode returns the ID/Value of this Error
+// ErrorCode returns the ID/Value of this Error.
 func (e Error) ErrorCode() ErrorCode {
 	return e.Code
 }
 
 // Error returns a human readable representation of the error.
 func (e Error) Error() string {
-	return fmt.Sprintf("%s: %s",
```

Comment:
The problem with this is that unless we add special logic Docker will show things like:
```
$ docker build tmp
Sending build context to Docker daemon 40.06 MB
Step 0 : FROM ubuntu
 ---> 07f8e8c5e660
Step 1 : ONBUILD onbuild
chainonbuild: Chaining ONBUILD via `ONBUILD ONBUILD` isn't allowed
```
and that "chainonbuild:" text is not only a change to existing Docker but not a very useful one.  Yes its might be nice for us (docker devs) but I don't think it does anything for the end user.  If we have cases where we want to show it then let's create a different func that we can call that returns this format.